homeassistant/components/airthings/sensor.py

```diff
@@ -160,7 +172,12 @@ def native_value(self) -> StateType:
     @property
     def available(self) -> bool:
         """Check if device and sensor is available in data."""
+        current_epoch_time = int(time.time())
         return (
             super().available
             and self.entity_description.key in self.coordinator.data[self._id].sensors
+            # Most AirThings sensors collect data at interval of max 10 minutes.
+            # If we don't hear from them for 60 minutes they're offline.
+            and current_epoch_time - self.coordinator.data[self._id].sensors.get("time")
+            < 3600
```

Comment:
I know some users use the Airthings cloud integration for BLE connected devices. In that case these sensors will be unavailable between every time the user syncs data using the Airthings app. It might just be an edge case... The best would might be to only do this for hub (smartlink) connected devices. Not sure if that's possible with the current implementation. Will look into this later this week